stable/ark/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.10.1
 description: A Helm chart for ark
 name: ark
-version: 3.0.0
+version: 3.0.1
```

Comment:
Seems like this is a breaking change which needs a major version bump.

Reply:
Agree, changed!